GundaGunde/GG00044.xml

```diff
@@ -1251,7 +1251,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         <desc><ref type="authFile" corresp="AT1036MercyCovenant"/></desc>
                      </decoNote>
                      </decoDesc>
-                     
+                     <additions>
+                        <list>
+                             <item xml:id="e1">
+                                <desc> In each incipits and explicits, the name <persName
+                                   ref="PRS14029Eraqlis"/> is provided as the copiyst name of the ʾAkkonu Bǝʾǝsi collection of the codicological unit. </desc>
```

Comment:
If he is clearly attested in the manuscript as scribe, he can be encoded `<persName ref="PRS14029Eraqlis" role="scribe"/>`